docs/reference/remap/strip_whitespace.cue

```diff
@@ -0,0 +1,43 @@
+package metadata
+
+remap: functions: strip_whitespace: {
+	arguments: [
+		{
+			name:        "value"
+			description: "The string to trim."
+			required:    true
+			type: ["string"]
+		},
+	]
+	return: ["string"]
+	category: "text"
+	description: #"""
+		Trims the whitespace from the start and end of the string. [Whitespace](https://en.wikipedia.org/wiki/Unicode_character_property#Whitespace) is any unicode character with the property `"WSpace=yes"`.
+		"""#
+	examples: [
+		{
+			title: "Success"
+			input: {
+				text: #"  A sentence.  "#
+			}
+			source: #"""
+				.trimmed = strip_whitespace(.text)
+				"""#
+			output: {
+				slice: "A sentence."
+			}
+		},
+		{
+			title: "Error"
+			input: {
+				text: 42
+			}
+			source: #"""
+				.trimmed = strip_whitespace(.text)
+				"""#
+			output: {
+				error: remap.errors.ArgumentError
+			}
+		},
```

Comment:
I feel like these errors are very repetitive, and don't really add any value to the documentation.

We already describe the accepted input types, we should just have a section in the general remap docs that describe what it means for a function to accept specific values, and what happens if you provide something that isn't acceptable.